### ISSUE TITLE
StarScreen won't show

### ISSUE THREAD

**Issue by AmineSmida** [fluentribbon/Fluent.Ribbon#394]
StarScreen won't show

App.xaml
```
<Application x:Class="SartScreenRibbon.App"
             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:local="clr-namespace:SartScreenRibbon"
             StartupUri="MainWindow.xaml">
    <Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <ResourceDictionary Source="pack://application:,,,/Fluent;Component/Themes/Office2013/Generic.xaml" />
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </Application.Resources>
</Application>
```

MainWindow.Xaml
```

<Fluent:RibbonWindow x:Class="SartScreenRibbon.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:SartScreenRibbon"
        xmlns:Fluent="urn:fluent-ribbon"
        mc:Ignorable="d"
        Title="MainWindow" Height="350" Width="525">
    <Grid>
        <Fluent:Ribbon AutomaticStateManagement="true" VerticalAlignment="Top" HorizontalAlignment="Stretch"  x:Name="ribbon">

            <Fluent:Ribbon.Menu>
                <Grid>
                    <!--Backstage items can be keytipped-->
                    <Fluent:Backstage>
                        <Fluent:BackstageTabControl>
                            <Fluent:Button Header="Save"
                                           KeyTip="S" />
                            <Fluent:Button Header="Save As"
                                           KeyTip="A"
                                           CanAddToQuickAccessToolBar="False"
                                            />
                            <Fluent:BackstageTabItem Header="TEST"
                                                     KeyTip="T"
                                                     Content="Hello">
                                <Fluent:BackstageTabItem.ContentTemplate>
                                    <DataTemplate>
                                        <Grid>
                                            <TextBox Text="{Binding Mode=OneWay}"
                                                     Width="6000" />
                                        </Grid>
                                    </DataTemplate>
                                </Fluent:BackstageTabItem.ContentTemplate>
                            </Fluent:BackstageTabItem>

                            <Fluent:BackstageTabItem Header="Change theme with fluent controls"
                                                     Content="Hello">
                                <Fluent:BackstageTabItem.ContentTemplate>
                                    <DataTemplate>
                                        <StackPanel>
                                        </StackPanel>
                                    </DataTemplate>
                                </Fluent:BackstageTabItem.ContentTemplate>
                            </Fluent:BackstageTabItem>

                            <Fluent:BackstageTabItem Header="Change theme with standard controls"
                                                     Content="Hello">
                                <Fluent:BackstageTabItem.ContentTemplate>
                                    <DataTemplate>
                                        <StackPanel>
                                            <TextBlock Text="{Binding RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type Fluent:BackstageTabControl}}, Path=ParentBackstage.ActualHeight}" />
                                        </StackPanel>
                                    </DataTemplate>
                                </Fluent:BackstageTabItem.ContentTemplate>
                            </Fluent:BackstageTabItem>

                            <Fluent:SeparatorTabItem Header="Separator" />

                            <Fluent:BackstageTabItem Header="Info"
                                                     KeyTip="I"
                                                     IsEnabled="False">
                                <ScrollViewer>
                                    <Grid Height="400"
                                          Background="Red">
                                        <ListBox Width="100"
                                                 Height="100"
                                                 VerticalAlignment="Top"
                                                 HorizontalAlignment="Left">
                                            <ListBoxItem>1</ListBoxItem>
                                            <ListBoxItem>2</ListBoxItem>
                                            <ListBoxItem>3</ListBoxItem>
                                            <ListBoxItem>4</ListBoxItem>
                                            <ListBoxItem>5</ListBoxItem>
                                            <ListBoxItem>6</ListBoxItem>
                                            <ListBoxItem>7</ListBoxItem>
                                            <ListBoxItem>8</ListBoxItem>
                                            <ListBoxItem>9</ListBoxItem>
                                            <ListBoxItem>10</ListBoxItem>
                                        </ListBox>
                                        <Fluent:ComboBox SelectedIndex="1">
                                            <ComboBoxItem>1</ComboBoxItem>
                                            <ComboBoxItem>2</ComboBoxItem>
                                            <ComboBoxItem>3</ComboBoxItem>
                                            <ComboBoxItem>4</ComboBoxItem>
                                            <ComboBoxItem>5</ComboBoxItem>
                                        </Fluent:ComboBox>
                                    </Grid>
                                </ScrollViewer>
                            </Fluent:BackstageTabItem>

                            <Fluent:BackstageTabItem Header="Recent"
                                                     KeyTip="R">
                                <TabControl Margin="20,5,20,5"
                                           >
                                    <Fluent:SeparatorTabItem Header="Test1" />
                                    <TabItem Header="1">

                                    </TabItem>
                                    <TabItem Header="2">

                                    </TabItem>
                                    <TabItem Header="3">

                                    </TabItem>
                                    <Fluent:SeparatorTabItem Header="Test2" />
                                    <TabItem Header="4">

                                    </TabItem>
                                </TabControl>
                            </Fluent:BackstageTabItem>

                            <Fluent:SeparatorTabItem />


                            <Fluent:BackstageTabItem x:Name="BackStagePrint" Header="Print" KeyTip="P" >
                                <DockPanel LastChildFill="True">
                                    <StackPanel x:Name="PrintControlsContainer" Orientation="Vertical">
                                        <TextBlock Margin="40,30,40,15" Foreground="#FF444444" Text="Print" FontSize="40" FontFamily="Segoe UI Light"/>
                                        <StackPanel Orientation="Horizontal" >
                                            <Button Margin="40,0,0,0" x:Name="buttonPrint" HorizontalAlignment="Left" VerticalAlignment="Top" Height="80" Width="80"   >
                                                <StackPanel Orientation="Vertical">
                                                    <Rectangle Stroke="Red"  Width="32" Height="32" />
                                                    <TextBlock HorizontalAlignment="Center" Text="Print" />
                                                </StackPanel>
                                            </Button>
                                            <Fluent:Spinner Margin="8,0,0,0" VerticalAlignment="Top" HorizontalAlignment="Left" Value="1" InputWidth="70" Format="0" Header="Copies: "/>
                                        </StackPanel>
                                        <Grid>
                                            <Grid.ColumnDefinitions>
                                                <ColumnDefinition Width="*" />
                                                <ColumnDefinition Width="*" />
                                            </Grid.ColumnDefinitions>
                                            <TextBlock Margin="40,15,0,0" Grid.Column="0" HorizontalAlignment="Left" Text="Printer" FontSize="22" TextTrimming="CharacterEllipsis" Foreground="#FF2b579a" />
                                            <Image x:Name="PrinterStatus" Grid.Column="1"  Width="11" Height="11" >
                                                <Image.ToolTip>
                                                    <ToolTip Width="210">
                                                        <StackPanel Orientation="Vertical">
                                                            <TextBlock FontWeight="SemiBold">
                                                                <Run Text="Printer Status"/>
                                                            </TextBlock>
                                                            <TextBlock />
                                                            <StackPanel Orientation="Horizontal">
                                                                <TextBlock Margin="5,0,0,0" Text="Status: " />
                                                                <TextBlock Text="{Binding Path=PrinterStatus}" TextWrapping="Wrap" TextTrimming="CharacterEllipsis" />
                                                            </StackPanel>
                                                            <StackPanel Orientation="Horizontal">
                                                                <TextBlock Margin="5,0,0,0" Text="Type: " />
                                                                <TextBlock Text="{Binding PrinterName}" />
                                                            </StackPanel>
                                                            <StackPanel Orientation="Horizontal">
                                                                <TextBlock Margin="5,0,0,0" Text="Where: " />
                                                                <TextBlock Text="{Binding PrinterPort}" />
                                                            </StackPanel>
                                                            <TextBlock Margin="5,0,0,0" Text="Comment: " />
                                                            <TextBlock />
                                                        </StackPanel>
                                                    </ToolTip>
                                                </Image.ToolTip>
                                            </Image>
                                        </Grid>
                                        <Fluent:ComboBox IsSynchronizedWithCurrentItem="True" x:Name="pList" Margin="40,5,0,0" DataContext="{Binding}" FontSize="11"  Header="Printer">
                                            <Fluent:ComboBox.ToolTip>
                                                <ToolTip Width="Auto">
                                                    <StackPanel Orientation="Vertical">
                                                        <TextBlock FontWeight="SemiBold" Text="Printer Status"/>
                                                        <TextBlock />
                                                        <StackPanel Orientation="Horizontal">
                                                            <TextBlock Margin="5,0,0,0" Text="Status: " />
                                                            <TextBlock Text="{Binding Path=PrinterStatus}" TextWrapping="WrapWithOverflow" TextTrimming="CharacterEllipsis" />
                                                        </StackPanel>
                                                        <StackPanel Orientation="Horizontal">
                                                            <TextBlock Margin="5,0,0,0" Text="Type: " />
                                                            <TextBlock Text="{Binding Path=PrinterName}" />
                                                        </StackPanel>
                                                        <StackPanel Orientation="Horizontal">
                                                            <TextBlock Margin="5,0,0,0" Text="Where: " />
                                                            <TextBlock Text="{Binding Path=PrinterPort}" />
                                                        </StackPanel>
                                                        <TextBlock Margin="5,0,0,0" Text="Comment: " />
                                                        <TextBlock />
                                                    </StackPanel>
                                                </ToolTip>
                                            </Fluent:ComboBox.ToolTip>
                                            <Fluent:ComboBox.Menu>
                                                <Fluent:RibbonMenu>
                                                    <Fluent:MenuItem Header="Add Printer..."  />
                                                    <Fluent:MenuItem Header="Print to File" />
                                                </Fluent:RibbonMenu>
                                            </Fluent:ComboBox.Menu>
                                        </Fluent:ComboBox>
                                        <TextBlock Margin="25,5,10,0" TextAlignment="Right" FontSize="12" ><Hyperlink x:Name="PrinterProperties" Foreground="#FF2b579a" ><Run Text="Printer Properties"/></Hyperlink></TextBlock>
                                        <TextBlock Margin="40,15,0,0" Grid.Column="0" HorizontalAlignment="Left" Text="Settings" FontSize="22" TextTrimming="CharacterEllipsis" Foreground="#FF2b579a" />
                                        <Fluent:ComboBox x:Name="dsListMenu" Margin="40,5,0,0" IsTextSearchEnabled="True" ResizeMode="Vertical" MinWidth="40" SizeDefinition="Large" SelectedIndex="0">
                                            <Button Tag="Document" x:Name="PrintAllPages" HorizontalContentAlignment="Left" Background="White" MaxWidth="300" BorderBrush="White">
                                                <StackPanel Orientation="Horizontal">
                                                    <Rectangle Stroke="Red"  Width="32" Height="32" />
                                                    <StackPanel Orientation="Vertical">
                                                        <TextBlock Margin="5,0,0,0" Text="Print All Pages" />
                                                        <TextBlock Margin="5,0,0,0" Text="The whole thing" />
                                                    </StackPanel>
                                                </StackPanel>
                                            </Button>
                                            <Button Tag="Document" HorizontalContentAlignment="Left" Background="White" MaxWidth="300" BorderBrush="White">
                                                <StackPanel Orientation="Horizontal">
                                                    <Rectangle Stroke="Red"  Width="32" Height="32" />
                                                    <StackPanel Orientation="Vertical">
                                                        <TextBlock Margin="5,0,0,0" Text="Print Selection" />
                                                        <TextBlock Margin="5,0,0,0" Text="Just what you selected" />
                                                    </StackPanel>
                                                </StackPanel>
                                            </Button>
                                            <Button Tag="Document" HorizontalContentAlignment="Left" Background="White" MaxWidth="300" BorderBrush="White">
                                                <StackPanel Orientation="Horizontal">
                                                    <Rectangle Stroke="Red"  Width="32" Height="32" />
                                                    <StackPanel Orientation="Vertical">
                                                        <TextBlock Margin="5,0,0,0" Text="Print Current Page" />
                                                        <TextBlock Margin="5,0,0,0" Text="Just this page" />
                                                    </StackPanel>
                                                </StackPanel>
                                            </Button>
                                            <Button Tag="Document" HorizontalContentAlignment="Left" Background="White" MaxWidth="300" BorderBrush="White">
                                                <StackPanel Orientation="Horizontal">
                                                    <Rectangle Stroke="Red"  Width="32" Height="32" />
                                                    <StackPanel Orientation="Vertical">
                                                        <TextBlock Margin="5,0,0,0" Text="Custom Print" />
                                                        <TextBlock Margin="5,0,0,0" Text="Type specific pages, sections or ranges" TextTrimming="CharacterEllipsis" />
                                                    </StackPanel>
                                                </StackPanel>
                                            </Button>
                                            <Button Tag="Document Info" HorizontalContentAlignment="Left" Background="White"  MaxWidth="300" BorderBrush="White">
                                                <StackPanel Orientation="Vertical">
                                                    <TextBlock Margin="20,0,0,0" Text="Document Info" />
                                                    <TextBlock Margin="20,0,0,0" Text="List of properties, such as file name, author and title" TextTrimming="CharacterEllipsis" />
                                                </StackPanel>
                                            </Button>
                                        </Fluent:ComboBox>
                                    </StackPanel>
                                    <Rectangle DockPanel.Dock="Left" Margin="50,60,0,30" Width="1" Fill="Black" HorizontalAlignment="Left" />
                                </DockPanel>
                            </Fluent:BackstageTabItem>

                            <Fluent:Button Header="Exit"
                                           KeyTip="X"
                                           Command="{Binding ExitCommand}" />
                        </Fluent:BackstageTabControl>
                    </Fluent:Backstage>
                </Grid>
            </Fluent:Ribbon.Menu>


            <Fluent:Ribbon.StartScreen>
                <Fluent:StartScreen x:Name="startScreen" IsOpen="True" Shown="True" >
                    <Fluent:StartScreenTabControl>
                        <Fluent:StartScreenTabControl.LeftContent>
                            <StackPanel Orientation="Vertical">
                                <Label Content="Fluent.Ribbon"
                                       FontSize="48"
                                       Foreground="{DynamicResource BackstageFontBrush}" />
                            </StackPanel>
                        </Fluent:StartScreenTabControl.LeftContent>

                        <Fluent:StartScreenTabControl.RightContent>
                            <StackPanel Orientation="Vertical">
                                <TextBlock VerticalAlignment="Center"
                                           HorizontalAlignment="Center">You can close the start screen by either clicking the button below or by pressing ESC</TextBlock>
                                <Fluent:Button HorizontalAlignment="Center"
                                               IsDefinitive="True">Close start screen</Fluent:Button>
                            </StackPanel>
                        </Fluent:StartScreenTabControl.RightContent>
                    </Fluent:StartScreenTabControl>
                </Fluent:StartScreen>
            </Fluent:Ribbon.StartScreen>

            <Fluent:RibbonTabItem Header="File">
                <Fluent:RibbonGroupBox >
                    <Fluent:RibbonGroupBox Header="Menu">
                        <Fluent:Button Name="ShowStartScreen"
                                   Size="Middle"
                                   Click="ShowStartScreen_OnClick">
                            Show start screen
                        </Fluent:Button>
                    </Fluent:RibbonGroupBox>
                </Fluent:RibbonGroupBox>
            </Fluent:RibbonTabItem>
        </Fluent:Ribbon>
    </Grid>
</Fluent:RibbonWindow>
```

MainWindow.xaml.cs

```
using System.Windows;

namespace SartScreenRibbon
{
    /// <summary>
    /// Interaction logic for MainWindow.xaml
    /// </summary>
    public partial class MainWindow 
    {
        public MainWindow()
        {
            InitializeComponent();
        }

        private void ShowStartScreen_OnClick(object sender, RoutedEventArgs e)
        {
            this.startScreen.Shown = false;
            this.startScreen.IsOpen = true;
        }
    }
}
```

---
### Environment

- Fluent.Ribbon __v 4.0.3.394
- Theme 2013 
- Windows  10
- .NET Framework  `4.5.2

[SartScreenRibbon.zip](https://github.com/fluentribbon/Fluent.Ribbon/files/736666/SartScreenRibbon.zip)

`

**Comment by batzen** [comment]
It does not show because you set `Shown="True"` which causes the `StartScreen` to not show.

**Comment by CoolIOT** [comment]
I tried every thing, and i got nothing. 
I finally find a solution, is to switch to the next version of Fluent ribbon.
in the version 5.0.0.0 it s work fine 
and **thank you for all the time and effort you put in this project**  :)

**Comment by batzen** [comment]
Sorry, forgot to mention that the startscreen is broken in 4.0.3 and fixed in 5.0. 5.0 should be released soon.
